beacon-chain/rpc/endpoints.go

```diff
@@ -124,6 +124,7 @@ func (s *Service) rewardsEndpoints(blocker lookup.Blocker, stater lookup.Stater,
 			name:     namespace + ".BlockRewards",
 			middleware: []middleware.Middleware{
 				middleware.AcceptHeaderHandler([]string{api.JsonMediaType}),
+				middleware.AcceptEncodingHeaderHandler(),
```

Comment:
I'm not sure if this actually makes sense to be added on all of these endpoints...

Reply:
If i remember correctly it was just a request on blobs

Reply:
What's the downside of being able to receive zipped data?